tests/common/widgets/current-refinements/options.ts

```diff
@@ -146,7 +146,7 @@ export function createOptionsTests(
                 <span
                   class="ais-CurrentRefinements-categoryLabel"
                 >
-                  Samsung
+                  Apple
```

Comment:
do you know what caused this change?

Reply:
I don't know exactly why, but it comes from https://github.com/algolia/instantsearch/pull/5123/files#diff-6af3d106c335b51f0e85d49f808b248a9704cb12dab1472606b6e9a383d981b1R508-R511.

I haven't been able to dig into where the refinements value order change, but during setup the order is as expected, and it changes as soon as the index is initialized.

A potential guess is that before, `mergeSearchParameters` only received one argument, and this short-circuited the reduce operation inside because it doesn't have an init value). Now we always have 2+ parameters with the addition of `mainHelper.state`, so we go through the reduce steps that merge all the facets type.

Ultimately, this order is the correct one, as it seems current refinements wants to respect refinements values order (except numeric refinements): https://github.com/algolia/instantsearch/blob/master/packages/instantsearch.js/src/connectors/current-refinements/connectCurrentRefinements.ts#L315.

Reply:
sounds good! maybe let's split out this part of the PR in a separate commit then, so it shows up differently in the changelog

Reply:
Changes are now in https://github.com/algolia/instantsearch/pull/5875